contrib/slurm/slurmexecutor.py

```diff
@@ -0,0 +1,315 @@
+# This file is part of BenchExec, a framework for reliable benchmarking:
+# https://github.com/sosy-lab/benchexec
+#
+# SPDX-FileCopyrightText: 2007-2020 Dirk Beyer <https://www.sosy-lab.org>
+# SPDX-FileCopyrightText: 2024 Levente Bajczi
+# SPDX-FileCopyrightText: Critical Systems Research Group
+# SPDX-FileCopyrightText: Budapest University of Technology and Economics <https://www.ftsrg.mit.bme.hu>
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import logging
+import os
+import queue
+import re
+import subprocess
+import sys
+import tempfile
+import threading
+import time
+
+from benchexec import BenchExecException, tooladapter
+from benchexec.util import ProcessExitCode
+
+sys.dont_write_bytecode = True  # prevent creation of .pyc files
+
+WORKER_THREADS = []
+STOPPED_BY_INTERRUPT = False
+
+
+def init(config, benchmark):
+    tool_locator = tooladapter.create_tool_locator(config)
+    benchmark.executable = benchmark.tool.executable(tool_locator)
+    benchmark.tool_version = benchmark.tool.version(benchmark.executable)
+
+    logging.info("Using %s version %s.", benchmark.tool_name, benchmark.tool_version)
```

Comment:
I would suggest to remove this to keep the output of BenchExec in this mode and standard mode similar. Either we want this always or never, using SLURM doesn't make a difference here.